README.md

```diff
@@ -24,6 +24,30 @@ Contributing
 We encourage you to contribute to ArborX! Please check out the
 [guidelines](CONTRIBUTING.md) about how to proceed.
 
+Citing ArborX
+-------------
+If you publish work which mentions ArborX, please cite the following paper:
+
+```BibTeX
+@article{arborx2020,
+  author = {Lebrun-Grandi\'{e}, D. and Prokopenko, A. and Turcksin, B. and Slattery, S. R.},
+  title = {{ArborX}: A Performance Portable Geometric Search Library},
```

Comment:
Don't you need `{ }` for every upper case in the title?

Reply:
If we want to retain the upper case, yes. But I don't know if we do. And if you look at the ACM bibtex export, it does not use any of them, even for ArborX, which is wrong.

Reply:
@Rombur do you feel strongly about it?

Reply:
No that's fine with me.